WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ConfigurableBundleProductView.swift

```diff
@@ -52,29 +53,30 @@ struct ConfigurableBundleProductView: View {
                     .shimmering()
                     .renderedIf(viewModel.bundleItemViewModels.isEmpty && viewModel.loadProductsErrorMessage == nil)
                 }
+                .padding(.horizontal, insets: safeAreaInsets)
                 .background(Color(.listForeground(modal: false)))
             }
-            .background(Color(.listBackground))
             .ignoresSafeArea(.container, edges: [.horizontal, .bottom])
```

Comment:
nit: I think we need to modify this line so it doesn't ignore the bottom safe area:

```suggestion
            .ignoresSafeArea(.container, edges: [.horizontal])
```

The `safeAreaInset` modifier is supposed to increase the safe area for the view it's modifying, to avoid overlap. Because `ScrollView` ignores the bottom safe area, the save button is overlapping the `ScrollView` content even when scrolled as far down as possible. I noticed this especially in landscape:

![Simulator Screenshot - iPhone 14 Pro - 2023-11-24 at 08 56 34](https://github.com/woocommerce/woocommerce-ios/assets/8658164/6201d49e-353c-4e7b-845f-397d13360388)

Increasing the safe area will still allow the button to overlap the content while scrolling, but when you scroll to the bottom of the `ScrollView` all the content will be above the button.

Reply:
That said, other lists like `ProductSelectorView` place the button directly below the scroll view (rather than using `safeAreaInset`) so the content never scrolls behind the button. Not sure if we want to use the same approach here for consistency.

Reply:
Nice catch on the scrolling issue! And good point about putting the sticky bottom view in a VStack. I was following a few previous use cases in the code base to put the bottom view using `safeAreaInset(edge: .bottom)`, but other than the separation of code that makes it easier to read, I haven't thought of other advantages of using `safeAreaInset(edge: .bottom)` - I think @itsmeichigo suggested using a while ago and I might have missed some other reasons (I tried checking a few PRs on the files but couldn't find the original discussion). Putting the bottom view in the bottom safe area does have a risk of the scrolling issue like how I forgot to account for the bottom safe area for the scroll view before. I updated the UI to use the VStack approach in https://github.com/woocommerce/woocommerce-ios/pull/11260/commits/061de84b06f039d6ce76fc589f49a9b22755a519

Reply:
I believe the context of my suggestion was that we had a view structure containing a `ScrollView` with a button at the bottom of it. I suggested using`safeAreaInset(edge: .bottom)` in cases where we want to stick a button at the bottom of the view instead of having to scroll a `ScrollView` to the end to see it.

This works similarly as having a `VStack` containing a `ScrollView` and a button as Jaclyn's last update above, but there are a few pros:
- With `safeAreaInset` we can avoid another layer of nested closures (with the outer `VStack`). This makes code structure clearer.
- The modifier adjusts the insets of the contained `ScrollView` to show its content with respect to the size of the added view. This is intended for when you want to pin a view at an edge of a view.

A few things to note:
- When we use `safeAreaInset`, we need to set a background color to the pinned view. This is to avoid the original content to be visible below the pinned view.
- Rachel's suggestion to remove the `.bottom` option for `ignoresSafeArea` is correct too. I believe we added that option to be able to keep the scroll view content visible outside of the bottom safe area. Removing it while using `safeAreaInset` should keep the scroll view behaving properly.